.github/workflows/main.yml

```diff
@@ -45,9 +45,12 @@ jobs:
         with:
           go-version: '1.16'
       - name: Check license headers
+        # We need https://github.com/google/addlicense/pull/70#issuecomment-888099920 landed
+        # to disable testdata folers in the repo.
         run: |
-          go env -w GOFLAGS=-mod=mod
-          make all
-          set -e
-          addlicense -l apache -c 'Security Scorecard Authors' -v *
-          git diff --exit-code
+          echo "license check disabled. Re-enable as soon as possible."
```

Comment:
s/Re-enable as soon as possible./Will be reenabled soon

Reply:
ack